src/components/unified-header/left-nav.js

```diff
@@ -220,6 +220,7 @@ class LeftNav extends mixin(createComponent, initComponentBySearch) {
             const childItem = eventMatches(evt, this.options.selectorLeftNavNestedListItem);
             const hasChildren = eventMatches(evt, this.options.selectorLeftNavListItemHasChildren);
             const flyoutItem = eventMatches(evt, this.options.selectorLeftNavFlyoutItem);
+            const hasLinkItem = !(leftNavItem.querySelector(this.options.selectorLeftNavListItemLink) === undefined);
```

Comment:
`querySelector()` returns `null` if no matching element is found, so strict equal check against `undefined` would never be true.